angular/projects/spark-angular/src/lib/components/sprk-tabbed-navigation/sprk-tabbed-navigation.component.spec.ts

```diff
@@ -133,6 +176,25 @@ describe('SprkTabbedNavigationComponent', () => {
     expect(testPanel2.focus).toHaveBeenCalled();
   });
 
+  it('should return null if tab does not contain active class', () => {
+    const test = component.getActiveTabIndex(
+      [document.createElement('div')],
+      'hello',
+    );
+    expect(test).toBe(null);
+  });
+
+  it('should move focus into the active panel when tab is pressed and the target is not role=tab', () => {
+    testTab2.click();
+    jest.spyOn(testPanel2, 'focus').mockImplementation(() => {});
+    const event: Event = new Event('keydown');
+    event['keyCode'] = 9;
+    testElement.classList.add('sprk-c-Tabs__button');
+    testElement.setAttribute('role', 'test');
+    testElement.dispatchEvent(event);
```

Comment:
How come we don't have to call fixture.detectChanges(); here but we do in the below test after that event?

Reply:
You're right, we need to add this.